lib/sender.js

```diff
@@ -51,10 +65,92 @@ class Sender extends Writable {
     if (this._options.debug) {
       console.log('%s', converted.trim());
     }
-    return this._socket.write(converted, error => this._handleError(error,
-      callback))
+    const writeRes = this._socket.write(
+      converted, error => this._handleError(error, callback));
+    return this._relp.enabled ? writeRes && this._relp.txno : writeRes;
+  }
+
+  resend(message, txno, callback) {
+    if (!this._relp.enabled) {
+      callback(new Error('resend only supported with a RELP sender'));
+      return false;
+    }
+    const converted = this._convert(message, txno)
+    if (this._options.debug) {
+      console.log('%s', converted.trim());
+    }
```

Comment:
one liner here? It's just a debug option

Reply:
one liner it is 57aa7a8804c75faae5bcfa99262a617d948b8cf5